src/lib/statusPanelOptionsBuilder.ts

```diff
@@ -25,51 +30,51 @@ export const statusPanelOptionsBuilder = (builder: PanelOptionsEditorBuilder<Sta
       name: 'Cluster Name',
       description: '',
       defaultValue: '',
-      category: ['Panel Options'],
+      category: ['Status Panel options'],
       settings: { expandTemplateVars: true },
     })
     .addTextInput({
       path: 'clusterUrl',
       name: 'Cluster URL',
       description: '',
       defaultValue: '',
-      category: ['Panel Options'],
+      category: ['Status Panel options'],
       settings: { expandTemplateVars: true },
     })
     .addBooleanSwitch({
       path: 'clusterTargetBlank',
       name: 'Open Cluster URL in new tab',
       defaultValue: false,
-      category: ['Panel Options'],
+      category: ['Status Panel options'],
       showIf: ({ clusterUrl }) => !!clusterUrl,
     })
-    .addTextInput({
-      path: 'namePrefix',
-      name: 'Remove Prefix',
-      defaultValue: '',
-      description: 'A prefix to remove from the name (helpful when repeating panel over a template)',
-      category: ['Panel Options'],
-    })
+    // .addTextInput({
```

Comment:
reason for removing?

Reply:
not implemented / was not implemented in the upstream. I need to see if we really need this option at all.